plugins/quarkus-backend/src/scaffolder/actions/quarkus/createApp.ts

```diff
@@ -119,8 +119,11 @@ export const createQuarkusApp = () => {
             if (! ctx.input.values.starterCode) {
                 noCode = "true";
             }
+
+            const streamkey = buildStreamKeyFromVersion(ctx.input.values.quarkusVersion);
+
             const postData = {
-                streamKey: ctx.input.values.quarkusVersion ? ctx.input.values.quarkusVersion : 'io.quarkus.platform:3.9',
+                streamKey: streamkey ? streamkey : 'io.quarkus.platform:3.9',
```

Comment:
Why is `3.9` hard coded ?

Reply:
We should perhaps throw an exception if the QuarkusVersion is not empty instead of hard a value that we know will change with future quarkus releases

Reply:
Yes, it is indeed done [here](https://github.com/aureamunoz/backstage-playground/blob/f67696a46771ef894051f4261773413accc256d8/plugins/quarkus-backend/src/scaffolder/actions/quarkus/createApp.ts#L185), so the streamkey shouldn't be empty at this point.